Sources/ArcGISToolkit/Components/Form/FormView.swift

```diff
@@ -39,18 +39,25 @@ public struct FormView: View {
     }
     
     public var body: some View {
-        ScrollView {
-            if isEvaluating {
-                ProgressView()
-            } else {
-                VStack(alignment: .leading) {
-                    FormHeader(title: featureForm?.title)
-                        .padding([.bottom], elementPadding)
-                    ForEach(model.visibleElements, id: \.id) { element in
-                        makeElement(element)
+        ScrollViewReader { scrollViewProxy in
+            ScrollView {
+                if isEvaluating {
+                    ProgressView()
+                } else {
+                    VStack(alignment: .leading) {
+                        FormHeader(title: featureForm?.title)
+                            .padding([.bottom], elementPadding)
+                        ForEach(model.visibleElements, id: \.self) { element in
+                            makeElement(element)
+                        }
                     }
                 }
             }
+            .onChange(of: model.focusedElement) { focusedElement in
+                if let focusedElement {
+                    withAnimation { scrollViewProxy.scrollTo(focusedElement, anchor: .top) }
```

Comment:
If the `focusedElement` is set before the keyboard appears the scroll view won't scroll:


https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/3998072/061260da-5c22-4ad7-b072-f633d038d8f7

Reply:
This should be detached from the keyboard entirely.

After looking at that specific form I think the issue is that there isn't enough content in the form for it to actually be scrollable.

I'll need to think more about how to handle that scenario.

Reply:
> After looking at that specific form I think the issue is that there isn't enough content in the form for it to actually be scrollable.

I'll do some more testing. I didn't think about that scenario.

Reply:
Okay I've reworked this but it involved floating panel changes. Please re-test.